### ISSUE TITLE
InceptionV3's Conv2d_2a_3x3.bn.running_var is too small.

### ISSUE THREAD

**Issue by ypw-cortex** [pytorch/vision#648]
https://gist.github.com/ypw-cortex/a1ae92d7bcdedd274f6490b582edf8fe

The running_var of batchnorm should not be so small。
```
bn = model.Conv2d_2a_3x3.bn
bn.running_var[11], bn.running_var[13]
(tensor(1.8647e-14), tensor(3.7400e-14))
```

The equivalent weight of batchnorm should not be so large.
```
running_std = bn.running_var.sqrt() + bn.eps
eq_weight = bn.weight.data / running_std
eq_weight
tensor([  1.1262,   1.0825,   0.8164,   1.2763,   3.4662,   0.8985,   0.8461,
          1.2279,   0.3658,   2.4894,   0.8431, 999.8634,   0.3131, 999.8066,
          1.0949,   1.1219,   0.3374,   0.7440,   2.7890,   0.3605,   1.1327,
          0.5710,   0.5503,   0.6672,   0.6222,   0.3376,   0.4227,   0.9465,
          0.2870,   0.8486,   0.9414,   0.6925])
```
```
eq_weight[11], eq_weight[13]
(tensor(999.8634), tensor(999.8066))
```

The weight of the convolutional layer should not be so small too.
```
conv = model.Conv2d_2a_3x3.conv
weight = conv.weight.data.mean(dim=-1).mean(dim=-1).mean(dim=-1)
weight[11], weight[13]
(tensor(2.9088e-10), tensor(2.5552e-10))
```

The other layer is the correct equivalent weight.

```
bn = model.Conv2d_1a_3x3.bn
running_std = bn.running_var.sqrt() + bn.eps
weights = bn.weight.data / running_std
weights
tensor([1.7271, 2.6709, 1.7531, 1.0139, 1.2894, 1.0954, 6.3732, 2.8702, 3.4773,
        1.6405, 2.6034, 1.4568, 1.0404, 3.3769, 2.1268, 2.3463, 1.2395, 1.1246,
        4.8683, 4.0129, 2.3488, 3.3749, 1.9351, 2.2495, 3.9881, 0.8731, 2.5312,
        1.1476, 2.0904, 2.0541, 2.6836, 5.9335])
```

**Comment by fmassa** [comment]
Can you expand on why those values are not right? For InceptionV3, we converted the weights straight from TensorFlow, and they gave expected, see https://github.com/pytorch/vision/pull/101.

**Comment by ypw-cortex** [comment]
> Can you expand on why those values are not right? For InceptionV3, we converted the weights straight from TensorFlow, and they gave expected, see #101.

I am doing some int8 fixed-point work. When I quantize other models, they kept the accuracy very good, but InceptionV3 could not give the expected results, because some running var is too small to calculate. 

The range of  int8 is -128~127, but `model.Conv2d_2a_3x3.bn.running_var[11]` is 1.8647e-14, which means the equivalent weight of bn is very big. So the other values of 0 to 2 will be squeezed to a small range, and int8 is not enough to guarantee their accuracy.
